src/settings/FinePolicy/FinePolicyForm.js

```diff
@@ -80,8 +80,7 @@ class FinePolicyForm extends React.Component {
       : formatMessage({ id: 'ui-circulation.settings.finePolicy.createEntryLabel' });
 
     const footerPaneProps = {
-      pristine,
-      submitting,
+      isSaveButtonDisabled: pristine || submitting,
```

Comment:
The synonym of `pristine` is untouched it is strange that the button is disabled when the state of a `Form` is untouched.